core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -481,6 +481,15 @@ public class RestConfig extends AbstractConfig {
           + "Default is false.";
   protected static final boolean PROXY_PROTOCOL_ENABLED_DEFAULT = false;
 
+  public static final String ERROR_HANDLING_ENABLED_CONFIG = "error.handling.enabled";
```

Comment:
I'm wondering if this is the right name for the property.

What the config really does is SUPPRESS_STACK_TRACE_IN_RESPONSE so something similar to that might be more appropriate. What do you think?

Reply:
yep i think that is better. I was trying to play with both stack trace and error handling and ended up shortening to error handling. I will add the one you mentioned.